src/pages/Chat/index.jsx

```diff
@@ -10,6 +10,7 @@ import { useState } from 'react'
 import Layout from './Chat.layout'
 
 const Chat = () => {
+  // @ts-ignore
```

Comment:
to solve this one please check this solution @e-azocar found

https://github.com/Azordev/dasher-user/issues/161#issuecomment-973234512